shared/utils/__tests__/logic.spec.ts

```diff
@@ -1,32 +1,26 @@
 import { ObjectId } from 'bson-ext'
```

Comment:
I don't think we need ObjectId in shared file, `FormDto`'s _id is just a plain string, we can mock it ourselves.